go.mod

```diff
@@ -419,3 +419,5 @@ replace (
 
 // Exclude this version because the version has an invalid checksum.
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace github.com/elastic/elastic-agent-client/v7 => github.com/AndersonQ/elastic-agent-client/v7 v7.0.0-20240109062623-907af5c552fb
```

Comment:
Did you leave it on purpose? I don't think it's a good idea.

Reply:
thanks, my bad

Reply:
@AndersonQ it seems you haven't pushed the update for this line.

Reply:
done